single-line-diagram-core/src/main/java/com/powsybl/sld/svg/FeederInfo.java

```diff
@@ -44,11 +45,11 @@ public FeederInfo(String componentType, LabelDirection arrowDirection, String le
     }
 
     public FeederInfo(String componentType, double value) {
-        this(componentType, value, null);
+        this(componentType, value, 0, null);
     }
 
-    public FeederInfo(String componentType, double value, String userDefinedId) {
-        this(componentType, value > 0 ? LabelDirection.OUT : LabelDirection.IN, null, String.valueOf(Math.round(value)), userDefinedId);
+    public FeederInfo(String componentType, double value, int precision, String userDefinedId) {
+        this(componentType, value > 0 ? LabelDirection.OUT : LabelDirection.IN, null, precision == 0 || Double.isNaN(value) ? String.valueOf(Math.round(value)) : String.valueOf(Precision.round(value, precision)), userDefinedId);
```

Comment:
I would have used `String.format` instead to avoid importing `Precision`, but ok. You did this to avoid creating the `"%.2d"` string?